components/footer/_footer.scss

```diff
@@ -34,3 +34,18 @@
     }
   }
 }
+.footer--eq {
+    .footer__link {
+      float: left;
+      margin-right: 1.5rem;
+    }
+  .footer__license {
```

Comment:
This should be a modifier or a utility class:
Either `class="footer__license--border"` or `class="u-border-top"`